docs/source/gbd2019_models/causes/heart_failure/index.rst

```diff
@@ -155,12 +142,63 @@ Vivarium Modeling Strategy
 Scope
 +++++
 
-Heart failure incidence rate will be from the heart failure envelope DisMod model multiplied by the proportion of incidence that is due to all causes other than ischemic heart disease, hypertensive heart disease, and chronic obstructive pulmonary disease. Heart failure due to those three etiologies are included in the relevant cause. Individuals will then experience the EMR from the heart failure envelope model once they are in this state. They will receive the HF disability weights while in this state.  
+Heart failure incidence rate will be from the heart failure envelope DisMod model, which represents 
+heart failure from any cause. Individuals will then experience the EMR from the heart failure envelope 
+model once they are in this state. They will receive the HF disability weights while in this state.  
 
-Heart failure incidence should be modified by SBP and LDL-c levels as per the following age-specific pooled cohort analysis_.
+Heart failure incidence should be modified by SBP and LDL-c levels as per the following age-specific 
+pooled cohort analysis_.
 
 .. _analysis: https://www.jacc.org/doi/full/10.1016/j.jacc.2019.03.529
 
+The relative risks can be utilized by SBP/LDL-C group based on these tables: 
+
+.. list-table:: Relative risk of heart failure for SBP 
+   :widths: 5 5 20
+   :header-rows: 1
+
+   * - SBP Group 
+     - Relative Risk 
+     - Notes 
+   * - <120
+     - Reference group  
+     - 
+   * - 120-129 
+     - 1.27 (1.13, 1.43) 
+     - 
+   * - 130-139 
+     - 1.5 (1.3, 1.73) 
+     - 
+   * - 140+ 
+     - 1.76 (1.43, 2.17) 
+     - 
+
+
+.. list-table:: Relative risk of heart failure for LDL-C 
```

Comment:
Did Greg say today that we should not include a risk-outcome pair for LDL-C to HF?

Reply:
Yeah - just updated this to reflect his latest email.